### DISCUSSION TITLE
Sky sports cricket and fox 501

### DISCUSSION THREAD

**Discussion by MridulBatra1** [iptv-org/community#1021] [Q&A]
Hi can please find the iptv link for Sky sports cricket and Fox 501 channels.
Thanks

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).

**Reply by Asghar463** [reply]
Hi can please find the iptv link for Sky sports cricket and Fox 501 channels.
Thanks
